vote/src/vote_account.rs

```diff
@@ -144,11 +145,44 @@ impl VoteAccounts {
         Some(vote_account)
     }
 
-    pub fn insert(&mut self, pubkey: Pubkey, (stake, vote_account): (u64, VoteAccount)) {
-        self.add_node_stake(stake, &vote_account);
+    pub fn insert(
```

Comment:
```suggestion
    pub fn modify_or_insert(
```
I'm not set on that name, but should be something that reflects the new behavior.

Reply:
I don't think I changed behavior? Insert was really upsert before too. It was a wrapper around HashMap::insert which inserts and returns the old value if any.